### ISSUE TITLE
Log events like motion, login attempts in bluecherry.log

### ISSUE THREAD

**Issue by antonsviridenko** [bluecherrydvr/bluecherry-apps-issues#142]
1) motion event
```
I(id/camera name): Motion detected
```
2) successful login event
```
I(): user "username" logged in from xx.xx.xx.xx
```
3) failed login attempt 
```
W(): failed login attempt for user "username" from xx.xx.xx.xx
```

**Comment by quixand** [comment]
It looks like you don't have a log handler in PHP, that would be a super-quick win for login auditing and any other GUI/ajax events

**Comment by antonsviridenko** [comment]
> It looks like you don't have a log handler in PHP, 

what exactly do you mean?

**Comment by quixand** [comment]
I had a look through the php in /usr/share/bluecherry/www (i think) I couldnt see anything handling logging. I don't have access to my test environment at the moment. its seems you have error and access logs from apache but thats it, please correct me if i'm wrong

**Comment by antonsviridenko** [comment]
Yes, that is, only apache access and error logs now. But syslog() is supported by PHP, I can log to bluecherry.log from PHP, I think there shouldn't be any problems

**Comment by quixand** [comment]
I've had some success with pear Log (https://pear.php.net/package/Log/) but on more recent projects i've gone to Monolog, its a bit more extendable and is the default in Symfony I think (https://github.com/Seldaek/monolog)
